app/src/main/java/org/keynote/godtools/android/activity/MainActivity.kt

```diff
@@ -138,8 +138,7 @@ class MainActivity :
     // endregion UI
 
     // region Feature Discovery
-    @JvmField
-    protected var featureDiscovery: TapTargetView? = null
+    private var featureDiscovery: TapTargetView? = null
```

Comment:
this wasn't related to analytics, just general code cleanup that accidentally got commit with the analytics changes